packages/web/webpack.config.js

```diff
@@ -8,6 +9,16 @@ const CopyPlugin = require('copy-webpack-plugin');
 const srcDir = 'src';
 const distDir = 'dist';
 
+const envDefaults = {
+  API_ORIGIN: 'https://staging.api.gradejs.com',
+  GA_ID: '',
+  PLAUSIBLE_DOMAIN: '',
+  DUMP_ANALYTICS: '',
+};
+const envPath = path.resolve(__dirname, '.env');
+const env = existsSync(envPath) ? dotenv.parse(readFileSync(envPath, { encoding: 'utf-8' })) : {};
```

Comment:
I would like to avoid parsing env files and having the default value for `API_ORIGIN`, since:
- Production building doesn't have any `.env` files. 
- dotenv can be used specifically for the development environment with a separate command in the `package.json` or inside `local_start.sh`
- Any deploy should trigger an error if the `API_ORIGIN` variable is not provided, just to avoid side-effects